merchant-app/app/src/main/java/com/googleinterns/smb/ConfirmationActivity.java

```diff
@@ -38,15 +38,17 @@ protected void onCreate(Bundle savedInstanceState) {
     private void initRecyclerView() {
         // get barcodes from launcher activity
         List<String> barcodes = getBarcodes();
-
+        if (barcodes.size() == 0) {
+            return;
+        }
```

Comment:
Can you add an issue, that we should land on Screen with some information text that no barcodes are available